src/User32.Desktop/User32+DeviceCotextValues.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DeviceContextValues"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        // <summary>
+        // Values to pass to the GetDCEx method.
+        // </summary>
+        [Flags]
+        public enum DeviceContextValues : uint
+        {
+            /// <summary>
+            /// DCX_WINDOW: Returns a DC that corresponds to the window rectangle rather than the client rectangle.
+            /// </summary>
+            Window = 0x00000001,
```

Comment:
Sorry, missed that from an old code I have been using. The newer ones are all on the format we use here to facilitate. Now it is fixed.